fl4health/strategies/feddg_ga_with_adaptive_constraint.py

```diff
@@ -0,0 +1,268 @@
+from logging import INFO, WARNING
```

Comment:
This class is needed in order to facilitate FedDG-GA for clients that also require adaptive constraint considerations. These are clients such as Ditto, MR-MTL, and FedProx. This strategy needs to coordinate the exchange of the loss weights in addition to the Generalization Adjustments for aggregation